en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,267 @@
+Manage assets with grunt
+========================
+
+Context
+-------
+
+To automate all the assets management process, the Open Orchestra Back Office uses the task manager
+``Grunt``_.
+
+*Grunt* is looking for its configuration inside the ``Gruntfile.js`` file in the root directory of
+the project. This file mainly parses external configuration files, split in grunt tasks and grunt
+tasks options.
+
+In order to simplify the management of external bundles assets, these bundles describe their own
+grunt task and/or task options. This way, when adding a new bundle to your project, all you have to do is
```

Comment:
I would rather propose this sentence : 

`As a result, adding a new bundle will require a little change in the ``Gruntfile.js` of your project to load the specific configuration.`